web/src/pages/projects/manage.tsx

```diff
@@ -27,11 +29,16 @@ export default function ManageProjectPage() {
   const [error, setError] = useState<string | null>(null);
   const [isApproving, setIsApproving] = useState(false);
   const [isSubmitting, setIsSubmitting] = useState(false);
+  const [showEditForm, setShowEditForm] = useState(false);
 
   const isApprovedResearcher = userData?.roles.includes("approved-researcher");
   const isAdmin = userData?.roles.includes("admin");
   const isTreOpsStaff = userData?.roles.includes("tre-ops-staff");
   const canApprove = isAdmin || isTreOpsStaff;
+  const canEdit = project?.creator_username === userData?.username; // TODO: what about study admins?
```

Comment:
maybe default to `true`? anyone that load the data will get a 403, so all good? (probably want TreOpsStaff to be able to edit(?)